src/common/pf_lldp.c

```diff
@@ -1061,7 +1036,14 @@ void pf_lldp_tx_restart (pnet_t * net, bool send)
 
 void pf_lldp_init (pnet_t * net)
 {
-   memset (&net->lldp_peer_info, 0, sizeof (net->lldp_peer_info));
+   uint16_t loc_port_num;
+   pf_port_t * p_port_data = NULL;
+
+   for (loc_port_num = 1; loc_port_num <= PNET_MAX_PORT; loc_port_num++)
```

Comment:
Use the port iterator to iterate over ports. See pf_port_init_port_iterator().

Reply:
Nice!

Reply:
Done